exporter/collector/metrics_test.go

```diff
@@ -65,8 +84,78 @@ func TestMetricToTimeSeries(t *testing.T) {
 		require.Same(t, ts[0].Resource, mr, "Should assign the passed in monitored resource")
 	})
 
+	t.Run("Sum without timestamps", func(t *testing.T) {
+		mapper, shutdown := newTestMetricMapper()
+		defer shutdown()
+		metric := pdata.NewMetric()
+		metric.SetDataType(pdata.MetricDataTypeSum)
+		sum := metric.Sum()
+		sum.SetIsMonotonic(true)
+		sum.SetAggregationTemporality(pdata.MetricAggregationTemporalityCumulative)
+		startTs := pdata.NewTimestampFromTime(start)
+		endTs := pdata.NewTimestampFromTime(start.Add(time.Hour))
+		// Add three points without start timestamps.
+		// The first one should be dropped to set the start timestamp for the rest
+		point := sum.DataPoints().AppendEmpty()
+		point.SetDoubleVal(10)
+		point.SetTimestamp(startTs)
+		point = sum.DataPoints().AppendEmpty()
+		point.SetDoubleVal(15)
+		point.SetTimestamp(endTs)
+		point = sum.DataPoints().AppendEmpty()
+		point.SetDoubleVal(16)
+		point.SetTimestamp(endTs)
+
+		ts := mapper.metricToTimeSeries(
+			mr,
+			labels{},
+			metric,
+		)
+		require.Len(t, ts, 2, "Should create one timeseries for each sum point")
+		require.Same(t, ts[0].Resource, mr, "Should assign the passed in monitored resource")
+		require.Equal(t, ts[0].Points[0].Value.GetDoubleValue(), 5.0, "Should normalize the resulting sum")
```

Comment:
nit check also the start timestamp gets normalized?

Reply:
done